Index: librofl-0.11.1/src/rofl/common/cthread.cpp
===================================================================
--- librofl-0.11.1.orig/src/rofl/common/cthread.cpp	2017-05-30 00:53:45.000000000 +0200
+++ librofl-0.11.1/src/rofl/common/cthread.cpp	2017-05-30 01:55:41.694030999 +0200
@@ -30,7 +30,7 @@
   }
 
   // eventfd
-  event_fd = eventfd(0, EFD_NONBLOCK);
+  event_fd = eventfd(0, 0);
   if (event_fd < 0) {
     throw eSysCall("eSysCall", "eventfd", __FILE__, __FUNCTION__, __LINE__);
   }
