Index: librofl-0.11.1/examples/ethswctld/cunixenv.h
===================================================================
--- librofl-0.11.1.orig/examples/ethswctld/cunixenv.h	2017-05-30 00:53:45.000000000 +0200
+++ librofl-0.11.1/examples/ethswctld/cunixenv.h	2017-05-30 01:56:34.722030999 +0200
@@ -6,7 +6,6 @@
 #define CUNIXENV_H 1
 
 #include <errno.h>
-#include <execinfo.h>
 #include <getopt.h>
 #include <iostream>
 #include <signal.h>
